components/registry/org.wso2.carbon.registry.security.ui/src/main/java/org/wso2/carbon/registry/security/vault/ui/PropertiesServiceClient.java

```diff
@@ -182,7 +182,7 @@ public void setProperty(HttpServletRequest request) throws Exception {
 	}
 
 	public void updateProperty(HttpServletRequest request) throws Exception {
-		String path = SecureVaultConstants.SYSTEM_CONFIG_CONNECTOR_SECURE_VAULT_CONFIG;
+		String path = SecureVaultConstants.ENCRYPTED_PROPERTY_CONFIG_REGISTRY_PATH;
 		String name = (String) Utils.getParameter(request, "name");
 		String value = (String) Utils.getParameter(request, "value");
 		String oldName = (String) Utils.getParameter(request, "oldName");
```

Comment:
"name" and "value" are duplicate in setProperty() and updateProperty() methods, try to define constants instead of string literals.